addons/armaos/functions/fnc_computer_standby.sqf

```diff
@@ -10,23 +10,14 @@
 
 params ["_computer"];
 
-private _standbyTime = 3;
+[_computer] spawn {
+	params ["_computer"];
 
-[
-	_standbyTime,
-	[_computer], 
-	{
-		params ["_args", "_elapsedTime", "_totalTime", "_errorCode"];
-		
-		private _computer = _args select 0;
+	sleep 3;
```

Comment:
If you disable the progessBar for the standby action, why using sleep command? If there is no progess, the sound could play immediately.

Reply:
fixed

Reply:
Ok. I think I understand now. You removed the progess bars completely. I presume you done that due to immersion reasons because the display on the laptop already gives a visual indicator. So the time is the previous time of the progress bar.

Reply:
I thought that we agreed on that in the past. The arguments for the removal were, that its quiete annoying to wait and watch the progress bar for 15s  and that in rl, you also dont have to hug your computer for the time it is booting.